packaging/windows/build_pypi_pkg_download_proxy/setup.py

```diff
@@ -14,18 +14,23 @@
 
 install_requires = [
    'pip',
+   'future',
+   'numpy',
+   'typing_extensions',
+   'dataclasses; python_version < "3.7"'
 ]
 
 class install_torch(setuptools.command.install.install):
     def run(self):
-        if sys.platform == 'win32' and sys.maxsize.bit_length() == 31:
-            raise UserWarning("We don't support Python x86." \
-                "Please install Python x64 instead.")
-
-        raise UserWarning(
-            f"Can not download torch binary from {torch_download_url}." \
-            f"Please visit {torch_download_url} for more details."
-        )
+        if sys.platform == 'win32':
+            if sys.maxsize.bit_length() == 31:
+                raise UserWarning("We don't support Python x86." \
+                    "Please install Python x64 instead.")
+        else:
+            raise UserWarning(
+                f"Can not download torch binary from {torch_download_url}." \
+                f"Please visit {torch_download_url} for more details."
```

Comment:
Perhaps we should point them towards https://pytorch.org/get-started/locally/ instead so they can grab the installation instructions from there

Reply:
Good idea!